README.md

```diff
@@ -19,7 +19,8 @@ Building from Source
 --------------------
 
 - You will need to install [sbt](https://www.scala-sbt.org/) if you don't already have it installed
-- You should have a Java Runtime installed. Java 8 minimum.
+- The Java Development Kit (JDK) used for running sbt should be Java 11 at minimum.
```

Comment:
Thanks - this is a good start. One nit - could you put sbt bit after the Java bit? In hindsight, sbt needs Java so we list Java first.